pkg/kubectl/cmd/resource/get.go

```diff
@@ -87,9 +87,8 @@ var (
 		namespace unless you pass --all-namespaces.
 
 		This command will hide resources that have completed, such as pods that are
```

Comment:
The advice here has always been wrong since it was only pods.  Remove the first sentence entirely please.

Reply:
@deads2k Modified.